source/npm/qsharp/ux/qsharp-ux.css

```diff
@@ -357,6 +357,14 @@ modern-normalize (see https://mattbrictson.com/blog/css-normalize-and-reset for
   fill: var(--vscode-button-hoverBackground, var(--nav-hover-background));
 }
 
+[data-vscode-theme-kind="vscode-high-contrast"] .bar {
```

Comment:
Note that "vscode-high-contrast" is for the dark HC theme. The light HC theme uses "vscode-high-contrast-light".